src/main/java/longah/commands/ResetCommand.java

```diff
@@ -0,0 +1,38 @@
+package longah.commands;
+
+import longah.exception.LongAhException;
+import longah.exception.ExceptionMessage;
+import longah.handler.PINHandler;
+import longah.node.Group;
+
+import java.util.Objects;
+
+public class ResetCommand extends Command {
+    /**
+     * Constructor for ResetCommand.
+     *
+     * @param commandString The command string.
+     * @param taskExpression The task expression.
+     */
+    public ResetCommand(String commandString, String taskExpression) {
+        super(commandString, taskExpression);
+    }
+
+    @Override
+    public void execute(Group group) throws LongAhException {
+        execute();
+    }
+
+    /**
+     * Executes the reset command.
+     *
+     * @throws LongAhException If unexpected additional parameters are found.
+     */
+    public void execute() throws LongAhException {
```

Comment:
uhh can u comment here on when this is called? `main` calls functions using a general call of `c.execute(group)` so not too clear on this

Reply:
called by using the command "reset password". "password" is passed in as an expression for checking